ApeWorX/ape
# feat: start ephemeral geth when using http plugin and web3 is not connected

### What I did

fixes: #204 

### How I did it

pygeth

### How to verify it

Add account to geth using an account from the test mnemonic:

```
    Account 0. 
    address: 0x1e59ce931B4CFea3fe4B875411e280e173cB7A9C
    private__key: 0xdd23ca549a97cb330b011aebb674730df8b14acaee42d211ab45692699ab8ba5
```

Create script

```python
from ape.cli import get_user_selected_account
from ape import project


def main():
    account = get_user_selected_account(prompt="Select an account to deploy 'Fund'")
    return account.deploy(project.Fund)
```

Run it (select your account you set above):

```bash
ape run deploy
```

It should start up an ephemeral geth and deploy the script.

I have also tested this a lot in my `ape test` branch.

### Checklist
